src/OpenLoco/src/Windows/IndustryList.cpp

```diff
@@ -275,6 +283,50 @@ namespace OpenLoco::Ui::Windows::IndustryList
             return rhsVar < lhsVar;
         }
 
+        static uint32_t getProductionLastMonth(const OpenLoco::Industry& industry, StringId* unitType = nullptr)
+        {
+            auto industryObj = ObjectManager::get<IndustryObject>(industry.objectId);
+            uint32_t cargoProduction = std::numeric_limits<uint32_t>::max();
+
+            if (unitType)
```

Comment:
you check `unitType` multiple times which is a bit of an anti-pattern - I would return early (ie the first line in this method) with `if (unitType == nullptr) return 0;`